desktop/README.md

```diff
@@ -26,6 +24,13 @@ export DEBUG='*'
 ./desktop/release/mac/WordPress.com.app/Contents/MacOS/WordPress.com
 ```
 
+### Development Mode
+
+For a more quicker development experience, the application can be run in "development mode", which executes the application code directly (i.e. without building the app). This development mode can be ran against production WordPress.com webapp, or a local instance of Calypso.
```

Comment:
Tiny suggestion. I'd remove `more` from the first sentence here. Leaving it as "For a quicker development experience,"